migrations/Version202406241409071101_taoQtiItem.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace oat\taoQtiItem\migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+use oat\taoQtiItem\scripts\install\EnableUuidQtiIdentifier;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ *
+ * phpcs:disable Squiz.Classes.ValidClassName
+ */
+final class Version202406241409071101_taoQtiItem extends AbstractMigration
```

Comment:
we don't need this migration, the default strategy should be current behavior of system

Reply:
This migration is modifying current behavior of system to apply new identifier generation strategy

Reply:
yes, and that is what we need to avoid, the new strategy should be applied only explicitly in specific environments, and shouldn't be applied automatically with update

Reply:
This is why it is used with feature flag. If feature flag is not set it will not change anything

Reply:
I changed to dynamic configuration using FeatureFlagConfigSwitcher